exercises/practice/triangle/triangle_test.gd

```diff
@@ -0,0 +1,132 @@
+## Equilateral triangle tests
+
+func test_equilateral_all_sides_are_equal(solution_script):
+    var args = [2, 2, 2]
+    var expected = true
+	return [expected, solution_script.is_equilateral(args)]
```

Comment:
The indentation is inconsistent here, so the tests will not be executed. Also, could you please use tabs everywhere, instead of spaces? :)

Reply:
Yeah, I haven’t gotten a chance to look at this one yet. VSCode is defaulting to spaces every time so I’ll need to dig into this. It’s a single tab equivalent to four spaces, right?

Reply:
Yes, Godot Editor displays a single tab with a width of 4 spaces. It doesn't convert tabs to spaces though.

Reply:
If the test runner is sensitive to tabs vs. spaces, we should confirm how the web editor handles tabs being added. If it converts a tab to spaces automatically, students will have problems.

Reply:
Good point ... @ErikSchierboom do you know if we can configure the web editor to handle tabs? If not, I believe a consistent indentation with spaces only will work too, but this is not a standard in Godot Engine :(

Reply:
We also need to make sure the test runner works okay with students working offline where we can’t control whether they use spaces or tabs.